### DISCUSSION TITLE
Can mojo emit llvm?

### DISCUSSION THREAD

**Discussion by hcye2022** [modular/modular#2811] [Q&A]
I am sure this has been asked, but I could not find it in the forum. Anyway here it goes --- can mojo emit llvm source code, which can be re-targed to other platforms using llc?

**Comment by fbreuer** [comment]
I am interested in this as well. Related question: is it possible to have the mojo compiler emit the MLIR representation of modules/packages? Either at the very beginning before the various passes run or towards the end of the lowering process.
